pkg/securitypolicy/securitypolicyenforcer.go

```diff
@@ -44,11 +44,29 @@ type SecurityPolicyEnforcer interface {
 	EnforceDeviceUnmountPolicy(unmountTarget string) (err error)
 	EnforceOverlayMountPolicy(containerID string, layerPaths []string, target string) (err error)
 	EnforceOverlayUnmountPolicy(target string) (err error)
-	EnforceCreateContainerPolicy(sandboxID string, containerID string,
-		argList []string, envList []string, workingDir string, mounts []oci.Mount, privileged bool, noNewPrivileges bool) (EnvList, bool, error)
+	EnforceCreateContainerPolicy(
```

Comment:
we should really lock down this signature and try not to introduce breaking changes.

Reply:
(can probably wait)

Reply:
Yes, this is something we should discuss